index.html

```diff
@@ -181,6 +181,31 @@ <h4 class="setup-similar-title">Похожие персонажи</h4>
   </div>
 </template>
 
+<script>
+  var wizardWidth = 70;
+  var wizardSpeed = 3;
+  var fireballSize = 22;
+
+  var getFireballSpeed = function(left) {
+    if (left = true) {
+      return wizardSpeed = 5;
+    } else {
+      (left = !true) {
+        return wizardSpeed = 2;
+    }
+    return wizardSpeed;
+  };
+  var getWizardHeight = function() {
+    var height = 1.337 * wizardWidth;
```

Comment:
ты посчитала в одной переменной, а вернула в другой. Проще здесь можно вывести так:
```suggestion
    return 1.337 * wizardWidth;
```
это избавляет от лишней переменной и сразу выводит результат